tests_extractor.d

```diff
@@ -69,7 +69,7 @@ private:
     {
 
         // write the origin source code line
-        outFile.writefln("// Line %d", u.line);
+        outFile.writefln("# line %d", u.line > 2 ? u.line - 2 : 0);
```

Comment:
I don't know why you changed `u.line` to  `u.line > 2 ? u.line - 2 : 0`, but the original logic was correct.
`#line N` tells the compiler that the next line is the N-th line. So for example, the generated code for:
https://github.com/dlang/tools/blob/d0552fdd8658cd2ded50b86ea3d5d5f24b37b8a3/test/tests_extractor/ascii.d#L109-L117

should be:
```D
#line 109
unittest
{
    import std.ascii;

    assert( isAlphaNum('A'));
    assert( isAlphaNum('1'));
    assert(!isAlphaNum('#'));

    // N.B.: does not return true for non-ASCII Unicode alphanumerics:
    assert(!isAlphaNum('á'));
}